cassandra-statefulset.yaml

```diff
@@ -91,18 +84,17 @@ spec:
   volumeClaimTemplates:
   - metadata:
       name: cassandra-data
-      annotations:
-        volume.beta.kubernetes.io/storage-class: fast
     spec:
       accessModes: [ "ReadWriteOnce" ]
+      storageClassName: fast
```

Comment:
Is this a hard requirement?

I'm not 100% sure how this works but what I've seen is if a k8s cluster can't satisfy this requirement then the statefulset will never even deploy.